doc/api/fs.md

```diff
@@ -1577,13 +1577,13 @@ _Note: [`fs.watch()`][] is more efficient than `fs.watchFile` and
 `fs.unwatchFile`. `fs.watch` should be used instead of `fs.watchFile` and
 `fs.unwatchFile` when possible._
 
-## fs.write(fd, buffer, offset, length[, position], callback)
+## fs.write(fd, buffer[, offset[, length[, position]]], callback)
 <!-- YAML
 added: v0.0.2
 -->
 
 * `fd` {Integer}
-* `buffer` {String | Buffer}
+* `buffer` {Buffer}
```

Comment:
Yes, it wasn't ever implemented. The code ends up either [calling binding.writeBuffer](https://github.com/nodejs/node/blob/e8fadd8da5a2d073815ba25a3eea470285a0b178/lib/fs.js#L684) or [binding.writeString](https://github.com/nodejs/node/blob/e8fadd8da5a2d073815ba25a3eea470285a0b178/lib/fs.js#L699) based on whether the second parameter is a buffer, and those functions aren't polymorphic.